QtCollider/QtDownload.h

```diff
@@ -57,6 +57,6 @@ class QtDownload : public QObject {
 
 public Q_SLOTS:
     void downloadFinished();
-    void downloadProgress(qint64 recieved, qint64 total);
+    void downloadProgress(qint64 received, qint64 total);
```

Comment:
Just noting that `recieved` (sic) appears to be unused elsewhere in our codebase.